### PR TITLE
Dev s3

### PR DESCRIPTION
**Pull request by joeaero19**

### CODE REVIEW

File: frontend/react/Dockerfile [hunk 1]
```diff
@@ -13,43 +13,95 @@ CMD ["/bin/bash", "-c", "npm install && ./env.sh && cp ./env-config.js ./public/
 # => Build container
 FROM dev as builder
 
+<<<<<<< HEAD
```
Comment (dwhitecl) [inline frontend/react/Dockerfile:16]: looks like youve got a conflict here

Comment (joeaero19) [reply frontend/react/Dockerfile]: I resolved this conflict by taking out the entire dockerfile. It is no longer needed.

File: .jenkins/Jenkinsfile.dev [hunk 2]
```diff
@@ -201,6 +256,7 @@ pipeline {
         }
       }
     }
+
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.dev:259]: Can you remove this new line

File: .jenkins/Jenkinsfile.dev [hunk 3]
```diff
@@ -166,6 +220,7 @@ pipeline {
...
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.dev:223]: Can you remove this new line

File: .jenkins/Jenkinsfile.dev [hunk 4]
```diff
@@ -147,6 +200,7 @@ pipeline {
...
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.dev:203]: Can you remove this new line

File: .jenkins/Jenkinsfile.dev [hunk 5]
```diff
@@ -122,6 +158,7 @@ pipeline {
...
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.dev:161]: Can you remove this new line

File: .jenkins/Jenkinsfile.dev [hunk 6]
```diff
@@ -97,6 +132,7 @@ pipeline {
...
     stage("Deploy Data Layer") {
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.dev:136]: Can you remove this new line

Comment (mdial89f) [reply .jenkins/Jenkinsfile.dev:136]: iw sa copy/pasting this on new lines haha it seemed aggresive.  if you can clean up the new lines, sweet.

File: .jenkins/Jenkinsfile.master [hunk 7]
```diff
@@ -33,8 +33,17 @@ pipeline {
             jenkinsUtils.buildAndPushImageToEcr("data/postgres_deployer", "postgres_deployer", [env.BUILD_TAG, env.JOB_BASE_NAME])
             jenkinsUtils.buildAndPushImageToEcr("frontend/api_postgres", "postgres_django", [env.BUILD_TAG, env.JOB_BASE_NAME])
             jenkinsUtils.buildAndPushImageToEcr("frontend/api_sqlserver", "sqlserver_django", [env.BUILD_TAG, env.JOB_BASE_NAME])
-            jenkinsUtils.buildAndPushImageToEcr("frontend/react", "react", [env.BUILD_TAG, env.JOB_BASE_NAME])
           }
+          dir('frontend/react'){
+
+           sh '''
+           tar -zcvf cartsbuild.tar.gz build
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.master:40]: The logic to copy env.sh and .env must exist here too:

        # copy .env and env.sh into the archive so we can use them after we unzip
        cp env.sh build/env.sh
        cp .env build/.env

Comment (mdial89f) [reply .jenkins/Jenkinsfile.master:40]: needs to be there before you tar it up

File: .jenkins/Jenkinsfile.master [hunk 8]
```diff
@@ -99,6 +107,14 @@ pipeline {
               ]
             )
           }
+          dir('frontend/aws') {
+            env.S3_BUCKET_NAME = jenkinsUtils.terraformOutput(env.APPLICATION_BUCKET, env.BRANCH_NAME, "s3_bucket_name")
+          }
+            sh '''
+               aws s3 cp s3://$APPLICATION_BUCKET/artifacts/$BUILD_TAG/cartsbuild.tar.gz cartsbuild.tar.gz
+               tar -xvzf cartsbuild.tar.gz
+               aws s3 sync build s3://${S3_BUCKET_NAME}
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.master:116]: The logic to populate env-config.js with the right urls must happen before the sync

               #Populate the static archive with the API_POSTGRES_URL and the API_SQLSERVER_URL before you sync it to the host bucket

              cd build  && ./env.sh  && cd ..

File: .jenkins/Jenkinsfile.prod [hunk 9]
```diff
@@ -95,6 +95,14 @@ pipeline {
               ]
             )
           }
+          dir('frontend/aws') {
+            env.S3_BUCKET_NAME = jenkinsUtils.terraformOutput(env.APPLICATION_BUCKET, env.BRANCH_NAME, "s3_bucket_name")
+          }
+            sh '''
+               aws s3 cp s3://$APPLICATION_BUCKET/artifacts/$VERSION/cartsbuild.tar.gz cartsbuild.tar.gz
+               tar -xvzf cartsbuild.tar.gz
+               aws s3 sync build s3://${S3_BUCKET_NAME}
```
Comment (mdial89f) [inline .jenkins/Jenkinsfile.prod:104]: ditto as above... have to substitute correct urls prior to sync
               #Populate the static archive with the API_POSTGRES_URL and the API_SQLSERVER_URL before you sync it to the host bucket

              cd build  && ./env.sh  && cd ..

**Review by mdial89f — CHANGES_REQUESTED** [review]

**Review by mdial89f — APPROVED** [review]

**Pull request merged**